src/components/questions/AudioRecorder.tsx

```diff
@@ -178,15 +178,16 @@ const AudioRecorder = (props: {
             <Icon
               name="stop"
               size={48}
-              accessibilityLabel={'Stop spraakopname knop'}
+              accessible={true}
+              accessibilityLabel={'Stop spraakopname'}
               style={styles.icon}
               onPress={onStopRecord}
             />
           ) : (
             <Icon
               name="microphone"
-              accessibilityLabel={'Opnameknop'}
-              accessibilityHint={'Start spraakopname knop'}
+              accessible={true}
+              accessibilityLabel={'Start spraakopname'}
```

Comment:
Of overal knop gebruiken of overal niet

Reply:
@GewoonJaap testen of dit ook werkt op iOS

Reply:
Besproken tijdens call